packages/matchbox/src/components/Slider/styles.js

```diff
@@ -0,0 +1,126 @@
+import { tokens } from '@sparkpost/design-tokens';
+import { StyledSlider } from './Slider';
+
+export const slider = props => {
+  let cursor = 'pointer';
+
+  if (props.disabled) {
+    cursor = 'not-allowed';
+  }
+
+  return `
+    padding: ${tokens.spacing_600} 0;
+    position: relative;
+    cursor: ${cursor};
+  `;
+};
+
+export const rail = () => `
+  position: absolute;
+  left: ${tokens.spacing_0};
+  right: ${tokens.spacing_0};
+
+  height: ${tokens.spacing_100};
+  background: ${tokens.color_gray_300};
+  border-radius: ${tokens.borderRadius_200};
+  transition: ${tokens.motionDuration_medium} background;
+  ${StyledSlider}:hover & {
+    background: ${tokens.color_gray_400};
+  }
+`;
+
+export const track = props => {
+  let background = tokens.color_blue_700;
+
+  if (props.disabled) {
+    background = tokens.color_gray_600;
+  }
+
+  return `
+    position: absolute;
+    left: ${tokens.spacing_0};
+    height: ${tokens.spacing_100};
+    background: ${background};
+    border-radius: ${tokens.borderRadius_200};
+  `;
+};
+
+export const tick = props => {
+  let background = tokens.color_gray_300;
+
+  if (props.disabled && props.included) {
+    background = tokens.color_gray_600;
+  }
+
+  if (!props.disabled && props.included) {
+    background = tokens.color_blue_700;
+  }
+
+  return `
+    position: absolute;
+    height: ${tokens.spacing_200};
+    width: ${tokens.spacing_200};
+    border-radius: ${tokens.borderRadius_circle};
+    transform: translate(-2px, -2px);
+
+    background: ${background};
+    transition: ${tokens.motionDuration_medium} background;
+
+    user-select: none;
+  `;
+};
+
+export const tickLabel = () => `
+  position: absolute;
+  top: ${tokens.spacing_400};
+  left: 50%;
+  transform: translate(-50%, 0);
+
+  color: ${tokens.color_gray_300};
```

Comment:
I think these are supposed to be a bit lighter

![Screen Shot 2020-03-02 at 3 11 18 PM](https://user-images.githubusercontent.com/3903325/75713621-28f2b300-5c98-11ea-871b-9274366cedbd.png)

Reply:
You mean darker? looks like they're supposed to be `color_gray_700`